config/gulp/build.js

```diff
@@ -57,8 +57,8 @@ function buildUSWDSComponents() {
         new Error(
           `${gulpfile} does not exist! You need a newer version of USWDS; ` +
             `specifically, one that includes the following PR: ` +
-            `https://github.com/uswds/uswds/pull/2050`,
-        ),
```

Comment:
**Question (non-blocking):** Were these commas flagged by any kind of linting / prettier commands?

The reason I ask is because trailing commas were added to the prettier config to match ES17 standards and reduce lines changed on updates.

Super minor, but I'd lean towards keeping them to match USWDS code that now enables trailing commas for all lines